inference-engine/src/low_precision_transformations/src/fake_quantize.cpp

```diff
@@ -182,8 +182,10 @@ std::shared_ptr<opset1::FakeQuantize> FakeQuantizeTransformation::fuseElementwis
         return nullptr;
     }
 
+    const auto data = fq::getData(eltwise);
+    const size_t outputIdx = NetworkHelper::getParentOutputIndex(data, eltwise);
     std::shared_ptr<opset1::FakeQuantize> newFakeQuantize = as_type_ptr<opset1::FakeQuantize>(fakeQuantize->clone_with_new_inputs({
-        fq::getData(eltwise),
+        data->output(outputIdx),
```

Comment:
instead each usage place update, update function `getData`, please: return output

Reply:
will be fixed later